xml/chapter5/section5/subsection2.xml

```diff
@@ -540,8 +568,8 @@ function new_label_number() {
     return label_counter;
 }
 
-function make_label(name) {
```

Comment:
best not use "name" here because that word is reserved for names (variables, constants) in programs.